thai/guidelines.md

```diff
@@ -0,0 +1,144 @@
+แนวทางการรับมือโรคระบาดในระยะต้น
```

Comment:
@attapornu @aprossi @futtetennista Just a note that this file is not up to date with changelog 41